### PR TITLE
Formatted types/l* packages with dprint

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
> 👉 Partial implementation of https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/65993. If you're seeing this and are surprised it's happening, please read the discussion and give feedback! ❤️ 

Implements the dprint formatting changes on packages whose names start with `l`. We're splitting up the changes to make it a bit easier to apply & review them en masse - and, if we need later on, revert / bisect for issues. These changes generated with roughly:

```shell
git checkout dprint-onboarding -- .dprint.jsonc
time npx dprint fmt "types/l*/**/*.t*"
# Formatted 617 files.
# npx dprint fmt "types/l*/**/*.t*"  2.92s user 0.68s system 194% cpu 1.847 total
rm .dprint.jsonc
```

CI failures are due to [DefinitelyTyped-tools > feat: remove formatting rules rules from dtslint](https://github.com/microsoft/DefinitelyTyped-tools/pull/699) not being ~merged~ included in CI yet.

Includes:

* #66355

### CODE REVIEW

File: types/layui-src/Layui.d.ts [hunk 1]
```diff
@@ -1,381 +1,381 @@
...
+    /**
+     * 内置模块名和外置名称映射
+     */
+    modules: Layui.Modules;
+
+    // -----------------prototype method-------------------------------------↓↓↓↓
+
+    // https://www.layui.com/doc/base/infrastructure.html
+    /**
+     * 对象是否为泛数组结构，实例：	<br/>&nbsp;
```
Comment (DangerBotOSS) [inline types/layui-src/Layui.d.ts:42]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/layui-src/misc.d.ts [hunk 2]
```diff
@@ -1,3546 +1,3542 @@
...
+         * 不显示编辑器工具栏，一般用于隐藏默认配置的工具bar
+         */
+        hideTool?: string[];
+        /**
+         * 设定编辑器的初始高度
+         */
+        height?: number | string;
+        /**
+         * 设定图片上传接口，如：uploadImage: {url: '/upload/', type: 'post'},需要服务端返回：    <br/>&nbsp;
+         *   {	<br/>&nbsp;
```
Comment (DangerBotOSS) [inline types/layui-src/misc.d.ts:1186]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## lab ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lab@latest/))
was missing the following properties:
1. report
1. execute
1. coverage
1. leaks
## lambda-log ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lambda-log@latest/))
was missing the following properties:
1. console
1. addLevel
1. result
1. addListener
1. on

<details>
<summary>as well as these 13 other properties...</summary>
<p>once, removeListener, off, removeAllListeners, setMaxListeners, getMaxListeners, listeners, rawListeners, emit, listenerCount, prependListener, prependOnceListener, eventNames</p>
</details>

## lambda-tester ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lambda-tester@latest/))
was missing the following properties:
1. checkForResourceLeak
1. noVersionCheck
1. enableVersionCheck
1. isVersionCheck
1. strict

<details>
<summary>as well as these 1 other properties...</summary>
<p>mocks</p>
</details>

## language-tags ([<kbd>unpkg</kbd>](https://unpkg.com/browse/language-tags@latest/))
was missing the following properties:
1. types
## lasso ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lasso@latest/))
was missing the following properties:
1. createExtensionSet
1. isExtensionSet
1. defaultLasso
1. pageOptimizer
1. toString
## launchpad ([<kbd>unpkg</kbd>](https://unpkg.com/browse/launchpad@latest/))
was missing the following properties:
1. remotePreview
## lazy.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lazy.js@latest/))
was missing the following properties:
1. VERSION
1. noop
1. identity
1. equality
1. createWrapper

<details>
<summary>as well as these 14 other properties...</summary>
<p>Sequence, ArrayLikeSequence, ObjectLikeSequence, StringLikeSequence, StreamLikeSequence, GeneratedSequence, AsyncSequence, AsyncHandle, clone, deprecate, curry, curryRight, createCallback, createComparator</p>
</details>

## ldap-filters ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ldap-filters@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'ldap-filters'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
1. escapeChars
## ldapjs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ldapjs@latest/))
was missing the following properties:
1. Client
1. Server
1. persistentSearch
1. PersistentSearchCache
1. filters

<details>
<summary>as well as these 2 other properties...</summary>
<p>url, parseURL</p>
</details>

## leaflet-freedraw ([<kbd>unpkg</kbd>](https://unpkg.com/browse/leaflet-freedraw@latest/))
was missing the following properties:
1. defaultOptions
1. edgesKey
1. instanceKey
1. modesKey
1. notifyDeferredKey

<details>
<summary>as well as these 1 other properties...</summary>
<p>polygons</p>
</details>

## leaflet.wms ([<kbd>unpkg</kbd>](https://unpkg.com/browse/leaflet.wms@latest/))
was missing the following properties:
1. Layer
1. TileLayer
1. tileLayer
1. Overlay
## ledgerhq__hw-transport ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ledgerhq__hw-transport@latest/))
was missing the following properties:
1. TransportError
1. TransportStatusError
1. TransportError
1. TransportStatusError
## leo-profanity ([<kbd>unpkg</kbd>](https://unpkg.com/browse/leo-profanity@latest/))
was missing the following properties:
1. removeWord
1. addWord
1. getReplacementWord
1. sanitize
1. proceed

<details>
<summary>as well as these 3 other properties...</summary>
<p>badWordsUsed, getDictionary, addDictionary</p>
</details>

## lesgo ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lesgo@latest/))
was missing the following properties:
1. httpMiddleware
1. normalizeHttpRequestMiddleware
1. successHttpResponseMiddleware
1. errorHttpResponseMiddleware
1. normalizeSQSMessageMiddleware

<details>
<summary>as well as these 12 other properties...</summary>
<p>verifyJwtMiddleware, AuroraDbService, DynamoDbService, ElastiCacheService, ElasticsearchService, JwtService, LoggerService, S3Service, SQSService, cache, elasticsearch, crypto</p>
</details>

## less2sass ([<kbd>unpkg</kbd>](https://unpkg.com/browse/less2sass@latest/))
was missing the following properties:
1. file
1. includeMixins
1. convertMixins
1. convertFunctionUnit
1. convertExtend

<details>
<summary>as well as these 5 other properties...</summary>
<p>convertColourHelpers, convertTildaStrings, convertInterpolatedVariables, convertVariables, convertFileExtensions</p>
</details>

## level-codec ([<kbd>unpkg</kbd>](https://unpkg.com/browse/level-codec@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'level-codec'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
## level-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/level-js@latest/))
was missing the following properties:
1. binaryKeys
1. arrayKeys
1. destroy
## leveldown ([<kbd>unpkg</kbd>](https://unpkg.com/browse/leveldown@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'leveldown'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. destroy
1. repair
## libmime ([<kbd>unpkg</kbd>](https://unpkg.com/browse/libmime@latest/))
was missing the following properties:
1. config
1. isPlainText
1. hasLongerLines
1. getHex
1. splitMimeEncodedString

<details>
<summary>as well as these 3 other properties...</summary>
<p>encodeURICharComponent, safeEncodeURIComponent, Libmime</p>
</details>

## libxmljs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/libxmljs@latest/))
was missing the following properties:
1. libxml_debug_enabled
1. features
1. Comment
1. Text
## license-checker ([<kbd>unpkg</kbd>](https://unpkg.com/browse/license-checker@latest/))
was missing the following properties:
1. print
1. asTree
1. asSummary
1. asCSV
1. asMarkDown

<details>
<summary>as well as these 2 other properties...</summary>
<p>parseJson, asFiles</p>
</details>

## light-sdk ([<kbd>unpkg</kbd>](https://unpkg.com/browse/light-sdk@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'light-sdk'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
## lime-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lime-js@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'lime-js'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
1. Guid
1. Envelope
1. Reason
1. ReasonCodes

<details>
<summary>as well as these 14 other properties...</summary>
<p>ExternalAuthentication, ContentTypes, AuthenticationScheme, firstLineError, setTimeout, setInterval, clearTimeout, clearInterval, enroll, unenroll, active, setImmediate, clearImmediate, Lime</p>
</details>

## linkifyjs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/linkifyjs@latest/))
was missing the following properties:
1. scanner
1. parser
1. options
1. inherits
## live-server ([<kbd>unpkg</kbd>](https://unpkg.com/browse/live-server@latest/))
was missing the following properties:
1. server
1. watcher
1. logLevel
## loadable__webpack-plugin ([<kbd>unpkg</kbd>](https://unpkg.com/browse/loadable__webpack-plugin@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'loadable__webpack-plugin'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. The declaration doesn't match the JavaScript module 'loadable__webpack-plugin'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
## lockfile ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lockfile@latest/))
was missing the following properties:
1. filetime
## lodash-es ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lodash-es@latest/))
was missing the following properties:
1. commit
1. lodash
1. next
1. plant
1. thru

<details>
<summary>as well as these 13 other properties...</summary>
<p>toIterator, toJSON, value, valueOf, wrapperAt, wrapperChain, wrapperCommit, wrapperLodash, wrapperNext, wrapperPlant, wrapperReverse, wrapperToIterator, wrapperValue</p>
</details>

## logg ([<kbd>unpkg</kbd>](https://unpkg.com/browse/logg@latest/))
was missing the following properties:
1. formatRecord
1. addConsole
1. removeConsole
1. removeListener
1. removeAllListeners

<details>
<summary>as well as these 1 other properties...</summary>
<p>on</p>
</details>

## loggly ([<kbd>unpkg</kbd>](https://unpkg.com/browse/loggly@latest/))
was missing the following properties:
1. serialize
1. Loggly
1. Search
## lokijs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lokijs@latest/))
was missing the following properties:
1. deepFreeze
1. freeze
1. unFreeze
1. Resultset
1. Comparators
## long-timeout ([<kbd>unpkg</kbd>](https://unpkg.com/browse/long-timeout@latest/))
was missing the following properties:
1. Interval
## loopback-boot ([<kbd>unpkg</kbd>](https://unpkg.com/browse/loopback-boot@latest/))
was missing the following properties:
1. ConfigLoader
1. compile
1. execute
1. utils
1. addInstructionsToBrowserify

<details>
<summary>as well as these 5 other properties...</summary>
<p>ConfigLoader, compile, execute, utils, addInstructionsToBrowserify</p>
</details>

## loopback ([<kbd>unpkg</kbd>](https://unpkg.com/browse/loopback@latest/))
was missing the following properties:
1. version
1. registry
1. errorHandler
1. faviconFile
1. remoteMethod

<details>
<summary>as well as these 15 other properties...</summary>
<p>template, createModel, configureModel, findModel, getModel, getModelByType, createDataSource, memory, DataSource, Connector, Memory, Mail, Remote, GeoPoint, ValidationError</p>
</details>

## lowlight ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lowlight@latest/))
was missing the following properties:
1. lowlight
## luaparse ([<kbd>unpkg</kbd>](https://unpkg.com/browse/luaparse@latest/))
was missing the following properties:
1. version
1. defaultOptions
1. tokenTypes
1. errors
1. ast

<details>
<summary>as well as these 3 other properties...</summary>
<p>lex, write, end</p>
</details>

## lunr ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lunr@latest/))
was missing the following properties:
1. FieldRef
1. Set
1. idf
1. QueryLexer
1. QueryParser
## lynx ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lynx@latest/))
was missing the following properties:
1. sample
## lz4 ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lz4@latest/))
was missing the following properties:
1. MAGICNUMBER
1. MAGICNUMBER_BUFFER
1. EOS
1. EOS_BUFFER
1. VERSION

<details>
<summary>as well as these 7 other properties...</summary>
<p>MAGICNUMBER_SKIPPABLE, blockMaxSizes, extension, STATES, SIZES, utils, version</p>
</details>

## lzma-native ([<kbd>unpkg</kbd>](https://unpkg.com/browse/lzma-native@latest/))
was missing the following properties:
1. version
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8034dbfc67f02ea1ee0a3a39471f7894e0b23e68
</p>

**Review by sandersn — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
Ready to merge

**Pull request merged**